src/main/kotlin/com/hibob/kotlin_exercises/extensions_ex3.kt

```diff
@@ -0,0 +1,34 @@
+package com.hibob.kotlin_exercises
+
+import kotlin.math.pow
+
+// q1:
+//fun sum(list: List<Int>):Int {
+//    var result = 0
+//    for (i in list) {
+//        result += i
+//    }
+//    return result
+//}
+
+// Ans_1:
+fun List<Int>.sum() : Int{
```

Comment:
good! you can map{} use in the future, look for it in google

Reply:
Thanks!|
Why does it matter if I do sum vs using map? (its not a for loop, isn't it?)